Documentation/markdown/CMakeOptions.md

```diff
@@ -0,0 +1,218 @@
+# CMake Options for the TUM Open Infra Platform.
+This lists all CMake Options that can be set for the TUM Open Infra Platform project.
+
+## Overview of the CMake Options
+
+A CMake Option is visible in the CMake GUI as a box that can be ticked (or not). 
+If you have made changes and want to return to the default values, you will have to delete the CMake cache and configure again. Refer to the CMake "Option" command [here](https://cmake.org/cmake/help/latest/command/option.html).
```

Comment:
For the sake of easier version control, each sentence (or even part-sentence) should be in new line. Markdown (smilar to LaTeX) ignores line breaks and use double line breaks for paragraphs.

Reply:
Resolved: https://github.com/tumcms/Open-Infra-Platform/pull/111/commits/1aaa94e615843a71ac9ab32e5aa727db7240db98

Reply:
Please put each sentence on a new line.

Reply:
Resolved: https://github.com/tumcms/Open-Infra-Platform/pull/111/commits/d17b8ad179ba7675d8c0c20dd900007d7eb5c65a